cunumeric/deferred.py

```diff
@@ -1111,7 +1112,12 @@ def swapaxes(self, axis1: int, axis2: int) -> DeferredArray:
 
     # Convert the source array to the destination array
     @auto_convert([1])
-    def convert(self, rhs: Any, warn: bool = True) -> None:
+    def convert(
+        self,
+        rhs: Any,
+        warn: bool = True,
+        nan_op: Optional[int] = None,
```

Comment:
I think you can change this to `nan_op: CovertCode = ConvertCode.NOOP`

Reply:
bd361b6dc3a341df6a95f957f32321f70827ad38